christfellowshipchurch/web-app-v2
# Update Packages

### About
Ran "yarn outdated" to look for all of the versions of the packages used for the web app that were out of date. Manually updated any outdated versions to ensure all the tools and software that we use are running on the latest version. 

### Test Instructions
1. Go to the terminal inside of web-app-v2
2. Run command "yarn outdated"
3. Ensure that command returns "done" and does not list any other packages.

### Screenshots

<img width="443" alt="Screenshot 2023-06-07 at 9 06 54 AM" src="https://github.com/christfellowshipchurch/web-app-v2/assets/93110503/13fd8c54-d6b8-4381-8532-0b51c5f6f44f">

### Closes Tickets
[CFDP-2589](https://christfellowshipchurch.atlassian.net/browse/CFDP-2589?atlOrigin=eyJpIjoiYWNhNGNjMmQ5ODYzNDQ1ODk1OTJlOGY1ZjE0MGNjYTUiLCJwIjoiaiJ9)


[CFDP-2589]: https://christfellowshipchurch.atlassian.net/browse/CFDP-2589?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
